woocommerce/wc-admin
# Extract IE11 specific styles to IE stylesheet

Fixes #357 

Adds an IE specific stylesheet for components and extracts existing styles specific to IE11 to individual stylesheets.

Note that this does not extract one-liner fixes that can be applied across different browsers, but instead focuses on larger chunks of code where we should maintain separate styles for IE.

This PR also does not attempt to fix other existing issues in IE.

@Aljullu Are there any other pieces of IE specific styles you think should be pulled into this stylesheet?

### Detailed test instructions:

1.  Boot up IE11.
2.  Check that the ie.css stylesheet is loading and no styling regressions have occurred between this branch and master.
3.  Check that the ie.css stylesheet is not loading under non-IE browsers.
